src/Microsoft.DotNet.Interactive.Browser/PlaywrightKernelConnector.cs

```diff
@@ -153,7 +153,7 @@ public Task<Kernel> CreateKernelAsync(string kernelName, BrowserKernelLanguage l
                 {
                     context.Publish(new ValueInfosProduced(new KernelValueInfo[]
                     {
-                        new(":root", typeof(ILocator))
+                        new(":root", new FormattedValue(PlainTextFormatter.MimeType,"document root"),typeof(ILocator))
```

Comment:
I don't think types for the HTML kernel shared values makes sense yet.

Reply:
what should be the formatted type here?